dfir_rs/src/scheduled/context.rs

```diff
@@ -234,6 +234,7 @@ impl Context {
         for task in self.tasks_to_spawn.drain(..) {
             self.task_join_handles.push(tokio::task::spawn_local(task));
         }
+        self.tasks_to_spawn.shrink_to_fit();
```

Comment:
Won't this result in more allocations for future `request_task`s?

Reply:
I'll remove that, that was unrelated. But to answer, request_tasks rarely (never?) gets called during actual running